recipes/rotate/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "rotate" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/richarddurbin/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz"
+  sha256: "da8a3c848cc05d59b579c375824008dd2d533c28d8a436a6e4761312d2152906"
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("rotate", max_pin="x") }}
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+  host:
+    - zlib
+  run:
+    - zlib
+
+
+test:
+  commands:
+    - rotate 2>&1 | grep can
```

Comment:
why "grep can" ?

Reply:
hi @bgruening for this package, the help menu shows the whole output. To simply the output, i just did a grep. If this is not a recommended practice then , we can keep below as it as. Please advise 

```
rotate
rotate [options] <input fasta filename>
  -x <pos>		position to become new start [0]
  -rc               reverse complement after rotating [false]
  -s <string>       rotate to start of search string
  -m <n_mismatch>   maximum number of mismatches allowed in search [0]
  -o <outfilename>  output file [standard output]
use either -x and optionally -rc, or -s
use filename '-' for standard input
can read .fa.gz files as well as .fa files
```